<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateBill</class>
 <widget class="QMainWindow" name="CreateBill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(6, 155, 116, 255), stop:0.727273 rgba(0, 0, 0, 255));
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>25</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="logo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 700 24pt &quot;Segoe Print&quot;;
font: 9pt &quot;Segoe Print&quot;;
font: 22pt &quot;Segoe Script&quot;;
text-decoration: underline;
font: 36pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>BillX</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
</string>
        </property>
        <property name="text">
         <string>Bill Number :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 98));
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="Digital_clock">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
</string>
        </property>
        <property name="text">
         <string>Clock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="DateTime">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
</string>
        </property>
        <property name="text">
         <string>Date And Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Undo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Bill To :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 98));
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Address : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 98));
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Mobile Number :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 98));
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Product Code :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_productcode">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 98));
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Quantity : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_quantity">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 98));
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Price :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_value">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
